api/apps/api/src/modules/features/features/domain/feature.ts

```diff
@@ -0,0 +1,40 @@
+import { FeatureFragment } from './feature-fragment';
+import { CustomFeature } from './custom-feature';
+
+export enum FeatureTag {
+  Species = 'species',
+  Bioregional = 'bioregional',
+}
+
+export class Feature {
+  constructor(
+    private readonly id: string,
+    private readonly tag: FeatureTag,
+    private readonly fragments: FeatureFragment[],
+    private readonly name: string,
+  ) {}
+
+  split(byKey: string): CustomFeature[] {
+    if (this.tag !== FeatureTag.Bioregional) {
+      throw new Error(`Only Bioregional features can be split.`);
+    }
+
+    return this.fragments
+      .filter((fragment) => fragment.key === byKey)
+      .map(
+        (fragment) =>
+          new CustomFeature(
+            `${fragment.value}`,
+            this.id,
+            fragment.id,
+            `some-id-asigned-by-factory`,
```

Comment:
You pulled the rug under my feet here 😝 - indeed, although I think your initial implementation would be preferable *in general*, the current designs call for `<species> / <bioregion/bioregion subset>` (the designs call for `... in ...` but the docs explain why we single-handedly decided to use `/` instead) - so this should be the *value* of the key/value pair chosen for this subset.